gammapy/data/data_store.py

```diff
@@ -455,6 +457,336 @@ def check(self, checks="all"):
         checker = DataStoreChecker(self)
         return checker.run(checks=checks)
 
+    ##########################################################################################
+    # My contribution:
+
+    def _obscore_def(self):
+        """Generate the Obscore default table
+
+        In case the obscore standard changes, this function should be changed consistently
+
+
+        Returns
+        -------
+        Astropy Table length=0
```

Comment:
Why not using the VO table from astropy ? 
https://docs.astropy.org/en/stable/io/votable/index.html#building-a-new-table-from-scratch
It should better match what you need, for example missing values are supported
https://docs.astropy.org/en/stable/io/votable/index.html#missing-values

Reply:
I was evaluating your suggestion, I understand that VOtable is more focused on the format of the tables and its connection with the xml format. Here I propose something more general, which is the creation of a table that can be written in different formats and that is why it seemed appropriate to use astropy table. In the next commit you will see that the new function only returns an astropy table object and then the user can write it in the format they need.
Regarding missing values, astropy table also has ways to treat them with masks. But currently we prioritize keeping the dtype of each obscore field (eg 0.0 for an empty double precision field according to obscore).

Reply:
Indeed, the astropy method is not adapted, proposing here only one serialization type. One should be more generic and be able to serialise this table in FITS, yaml, xml, ...